### ISSUE TITLE
[MC 1.11] RuntimeException: Error while preparing NBT wrapper classes.

### ISSUE THREAD

**Issue by mibby** [goncalomb/NBTEditor#6]
@goncalomb Update needed to support Spigot 1.11. :)

```
[21:39:55] [Server thread/INFO]: [NBTEditor] Enabling NBTEditor v2.1-SNAPSHOT
[21:39:55] [Server thread/ERROR]: [NBTEditor] Error preparing reflection objects
java.lang.RuntimeException: Error while preparing NBT wrapper classes.
	at com.goncalomb.bukkit.mylib.reflect.NBTBase.prepareReflection(NBTBase.java:53) ~[nbteditor-2.1-SNAPSHOT.jar:?]
	at com.goncalomb.bukkit.nbteditor.NBTEditor.onEnable(NBTEditor.java:72) [nbteditor-2.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:271) [patched_1.11.jar:git-Paper-928]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:316) [patched_1.11.jar:git-Paper-928]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:407) [patched_1.11.jar:git-Paper-928]
	at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugin(CraftServer.java:377) [patched_1.11.jar:git-Paper-928]
	at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugins(CraftServer.java:327) [patched_1.11.jar:git-Paper-928]
	at net.minecraft.server.v1_11_R1.MinecraftServer.t(MinecraftServer.java:433) [patched_1.11.jar:git-Paper-928]
	at net.minecraft.server.v1_11_R1.MinecraftServer.l(MinecraftServer.java:394) [patched_1.11.jar:git-Paper-928]
	at net.minecraft.server.v1_11_R1.MinecraftServer.a(MinecraftServer.java:332) [patched_1.11.jar:git-Paper-928]
	at net.minecraft.server.v1_11_R1.DedicatedServer.init(DedicatedServer.java:276) [patched_1.11.jar:git-Paper-928]
	at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:599) [patched_1.11.jar:git-Paper-928]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_112]
Caused by: java.lang.NoSuchMethodException: net.minecraft.server.v1_11_R1.ItemStack.createStack(net.minecraft.server.v1_11_R1.NBTTagCompound)
	at java.lang.Class.getMethod(Unknown Source) ~[?:1.8.0_112]
	at com.goncalomb.bukkit.mylib.reflect.NBTUtils.prepareReflection(NBTUtils.java:65) ~[nbteditor-2.1-SNAPSHOT.jar:?]
	at com.goncalomb.bukkit.mylib.reflect.NBTBase.prepareReflection(NBTBase.java:51) ~[nbteditor-2.1-SNAPSHOT.jar:?]
	... 12 more
[21:39:55] [Server thread/ERROR]: [NBTEditor] This version of NBTEditor is not compatible with this version of Bukkit
```

**Comment by mibby** [comment]
Self compiling against 1.11. Still errors, but provides some info when compiling for what new things are missing. 

Missing enchants are BINDING_CURSE and VANISHING_CURSE.

```
Missing BoS entities: SKELETON_HORSE, ZOMBIE_HORSE, STRAY, ENDER_SIGNAL, PAINTING, WITHER_SKELETON, HUSK, ZOMBIE_VILLAGER, EVOKER, DONKEY, MULE, WEATHER, LINGERING_POTION, ITEM_FRAME, UNKNOWN, COMPLEX_PART, EVOKER_FANGS, ELDER_GUARDIAN, LLAMA_SPIT, PLAYER, VEX, SHULKER_BULLET, FISHING_HOOK, LEASH_HITCH, LLAMA, VINDICATOR, LIGHTNING.
```

**Comment by goncalomb** [comment]
Yes, those are just some information on what is missing.
Thanks for the info, I'll work on it over this weekend.

**Comment by goncalomb** [comment]
Just pushed some commits ([mc-1.11](https://github.com/goncalomb/NBTEditor/tree/mc-1.11)) to make it compatible with 1.11. This update requires a lot of changes, I'll do a proper release soon.

Horse types still need to be fixed, the new variables and entities are still missing.

Old BoSs (except Horses) should work fine. Just in case, make a backup with "pickblock" on creative mode or using `/itemstorage`, and then use the command `/bos refresh` to update them.

**Comment by goncalomb** [comment]
Everything should be compatible now (horses are fixed: 76f6e5d8f5672b45ebb1a6aefd96bd1a827cd070). The new stuff will be added soon.
